frontend/src/types.ts

```diff
@@ -1394,6 +1394,12 @@ export enum ChartDisplayType {
     BoldNumber = 'BoldNumber',
 }
 
+export const ChartDisplayTypesThatDoNotShowValuesOnSeries = [
```

Comment:
Should probably go into `src/lib/constants.tsx`, where we have `NON_TIME_SERIES_DISPLAY_TYPES`, `NON_BREAKDOWN_DISPLAY_TYPES`, etc.